### PR TITLE
Add atan support for missings

### PR DESCRIPTION
**Pull request by nicholasbalasus**
allowing atan to deal with two missing arguments

### CODE REVIEW

**Comment by oscardssmith** [comment]
This seems reasonable given that we already define `atan(missing)`.

**Comment by garrison** [comment]
Thank you for the pull request (the first ever submitted from your account, according to GitHub!).

- Is there any reason not to have it return `missing` as well if only one of the provided arguments is `missing`?
- Might it be worth adding test(s)?

File: base/math.jl [hunk 1]
```diff
@@ -1318,6 +1318,7 @@ for f in (:sin, :cos, :tan, :asin, :atan, :acos,
     end
     @eval $(f)(::Missing) = missing
 end
+atan(::Missing,::Missing) = missing
```
Comment (garrison) [inline base/math.jl:1321]: This line would be more consistent with the rest of the julia code base if a space were added after the comma.  (There are a handful of exceptions, the space is there much more often than it is not.)

Comment (nicholasbalasus) [reply base/math.jl]: Thank you, garrison! I have just added support for the cases where one of the values is missing and the other is not. I believe `::Number` is better for this than `::Any` but am not sure!

```
atan(::Missing, ::Missing) = missing
atan(::Number, ::Missing) = missing
atan(::Missing, ::Number) = missing
```

Comment (garrison) [reply base/math.jl]: I agree that `::Number` makes the most sense, as it makes this consistent with other two-argument math functions (e.g. `*`, `/`).

**Comment by garrison** [comment]
I think the easiest (and, in my opinion, the most appropriate) place to add a test case would be to add it to the [list of "arithmetic operators"](https://github.com/JuliaLang/julia/blob/361729957eecad3b8c7d0aa7d32dc48ac619ca13/test/missing.jl#L86).  If this path is chosen, I think the phrase "arithmetic operators" should perhaps be renamed to "binary operations" (i.e., within the variable name, testset name, and comments).  Note that the single argument form of `atan` tested [later in the same file](https://github.com/JuliaLang/julia/blob/361729957eecad3b8c7d0aa7d32dc48ac619ca13/test/missing.jl#L183), under "elementary functions".

**Comment by nicholasbalasus** [comment]
Added now (and it looks like the checks passed now after I had quite a few issues with the white space bot, my apologies!) 

as it’s written now, I left atan in the “elementary functions” test set as well for the one input test, though this could definitely be changed if it would be better practice to only have it in one place.

**Review by garrison — APPROVED** [review]

**Pull request merged**